### PR TITLE
Adds Zero API guide

### PR DESCRIPTION
**Pull request by ZPain8464**
This PR adds an API guide that shows users how to create an API User Token, exchange it for an ID token, and make their first authenticated request to the API. It shows users how to build a policy and attach it to a route. 

I used this [Stripe API guide](https://docs.stripe.com/get-started/api-request) as a reference.

I know that we're still undecided about the Zero information architecture, but this was a requested resource for the Zero GA. For now, I'd like feedback about the guide. If we decide to move this guide after we decide on a path forward with Zero docs, that's fine.

Related to https://github.com/pomerium/internal/issues/1760

### CODE REVIEW

**Comment by ZPain8464** [comment]
Some things to consider: 
- Should this include a section on onboarding with the Zero API? Or onboarding an invited user?
- Should it go beyond just adding a route and policy? For example, should it cover custom domains?
- We're updating the sidebar structure in another PR. This would still live under the Pomerium Zero sidebar slice.

File: content/docs/zero/api-guide.md [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+Copy your organization ID.
+
+### Get namespace ID
+
+In Pomerium Zero, each cluster is assigned its own namespace. To make changes to your cluster's configuration, you need your cluster's namespace ID.
+
+To get your cluster's namespace ID, send a `GET` request to the `/namespaces` endpoint, replacing `{organizationId}` with your own:
+
+```curl
+curl --location 'https://console.pomerium.app/api/v0/organizations/{organizationId}/namespaces' \
```
Comment (wasaga) [inline content/docs/zero/api-guide.md:96]:
```suggestion
curl --location 'https://console.pomerium.app/api/v0/organizations/{organizationId}/clusters' \
```

Maybe we should list clusters as this is the concept available in the API, while namespace is currently not part of the UI concepts; the cluster entity would have `namespaceId` that user could use in the subsequent calls to create routes and policies

File: content/docs/zero/api-guide.md [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+--header 'Authorization: Bearer <ID-TOKEN>'
+```
+
+You'll get a response similar to the example below. In the JSON payload, the first object contains data about the organization, which functions as a root namespace with admin-level permissions.
```
Comment (wasaga) [inline content/docs/zero/api-guide.md:100]:
```suggestion
You'll get a response similar to the example below. In the JSON payload, the first object contains data about the cluster, which has an ID of a root namespace other entities such as routes and policies are assigned to.
```

File: content/docs/zero/api-guide.md [hunk 3]
```diff
@@ -0,0 +1,250 @@
...
+            {
+            "is": "user@example.com"
+            }
+        }
+      ]
+    }
+  },
+  "description": "Only allow users access with a matching email address.",
+  "explanation": "You don'\''t have the correct email address to access this service.",
+  "remediation": "Email address must match user@example.com"
```
Comment (wasaga) [inline content/docs/zero/api-guide.md:157]:
```suggestion
  "remediation": "Use account credentials that match policy requirements."
```

let's not disclose the content of authorization policy

File: content/docs/zero/api-guide.md [hunk 4]
```diff
@@ -0,0 +1,250 @@
...
+      "and": [
+        {
+          "email": {
+            "is": "user@example.com"
+          }
+        }
+      ]
+    }
+  },
+  "remediation": "Email address must match user@example.com",
```
Comment (wasaga) [inline content/docs/zero/api-guide.md:190]:
```suggestion
  "remediation": "Use account credentials that match policy requirements.",
```

File: content/docs/zero/api-guide.md [hunk 5]
```diff
@@ -0,0 +1,250 @@
...
+    "name": "HTTP Bin",
+    "from": "https://httpbin.{CLUSTER_STARTER_SUBDOMAIN}.pomerium.app",
+    "to": ["http://httpbin:80"],
+    "allowWebsockets": false,
+    "allowSpdy": false,
+    "tlsSkipVerify": false,
+    "tlsUpstreamAllowRenegotiation": false,
+    "preserveHostHeader": false,
+    "enableGoogleCloudServerlessAuthentication": false,
+    "showErrorDetails": false,
```
Comment (wasaga) [inline content/docs/zero/api-guide.md:217]:
```suggestion
```

File: content/docs/zero/api-guide.md (hunk 1 above)
Comment (ZPain8464) [reply content/docs/zero/api-guide.md]: I'm incorporating this feedback and removing the previous payload example. I'll also remove the parent/child cluster relationship explanation.

**Review by wasaga — APPROVED** [review]

**Pull request merged**